frontend/lib/src/StreamlitEndpoints.ts

```diff
@@ -91,22 +91,8 @@ export interface StreamlitEndpoints {
   deleteFileAtURL?(fileUrl: string, sessionId: string): Promise<void>
 
   /**
-   * Fetch a cached ForwardMsg from the server.
-   *
-   * This is called when the ForwardMessageCache has a cache miss - that is, when
-   * the server sends a ForwardMsg reference and we don't have the original message
-   * in our local cache.
-   *
-   * @param hash the message's hash
-   *
-   * @return a Promise<Uint8Array> that resolves with the serialized ForwardMsg data returned
-   * from the server. Callers can use `ForwardMsg.decode` to deserialize the data.
-   */
-  fetchCachedForwardMsg(hash: string): Promise<Uint8Array>
-
-  /**
-   * setFileUploadClientConfig.
-   * @param config the object that contains prefix and headers object
+   * Set JWT Header.
```

Comment:
This `* Set JWT Header.` part has been removed. It probably appears this is just an artifact of auto-merge; we could remove it.

Reply:
Good catch 👍  Fixed it